### ISSUE TITLE
TryNormalize changed behavior

### ISSUE THREAD

**Issue by petterl** [GlobalPhone/GlobalPhone#28]
There is an update in behavior in GlobalPhone 5.0.2 -> 6.0.0.
Before an 
`_globalPhone.TryNormalize("#08-2 55 55", out normalized, "SE")`
returned **null**
Now it returns
**"+46825555"**

Is this intentional?

**Comment by wallymathieu** [comment]
The 6.0.0 is major release where the previous implementation was swapped for libphonenumber. This could have resulted in bugs and weirdness. So, no, the change of behaviour is unintentional. Is this a bug?

**Comment by petterl** [comment]
These numbers was never normalized before but are now:

- "-- 40 55"
- "08-2 55 55"
- "#08 640 555 -55"

These numbers was not normalized before and still not:

- "+4670722345623432432342"
- " "

If it is unintentional it might be considered a bug yes. But the normalized versions are still numbers that are cleaned so it might be correct depending on what you think.

**Comment by wallymathieu** [comment]
Indeed. @mbrrg can add you as an owner or whatnot of the GlobalPhone organization. He should also have nuget access to add you to the nugets if you want to contribute fixes/publish new versions.
